src/components/tooltip/Tooltip.tsx

```diff
@@ -25,6 +25,9 @@ export type TooltipProps = BaseProps & {
    * @default undefined
    */
   disabled?: boolean;
+
+  maxWidth?: number | string;
```

Comment:
According to your implementation below, number is invalid value for `maxWidth` since `maxWidth` in css requires unit of measurement (px, %, ...). Either removing the `number` type off of the union or implementing logic to check for when it's number, default the unit of measurement to `px` for example.
```suggestion
  maxWidth?: string;
```